STANDARD.md

```diff
@@ -222,20 +227,32 @@ Each file's *root* group (path `/`) must further define the attributes:
                    for `fileBased` formats the iteration must be included
                    in the file name;
                    the format depends on the selected `iterationEncoding` method
+    - note: it is not required that every openPMD iteration contains an update for each declared openPMD record (see below)
     - examples:
       - for `fileBased`:
         - `filename_%T.h5` (without file system directories)
-      - for `groupBased`:
+      - for `groupBased`: (fixed value)
         - `/data/%T/` (must be equal to and encoded in the `basePath`)
+      - for `stepBased`: (fixed value)
+        - data-format internal convention
+        - *slowest varying index* of data
+
+### `stepBased` Encoding of Iterations
+
+In order to correlate openPMD iterations with an index of data-format internal updates/steps or an index in the slowest varying dimension of an array, the *root* group (path `/`) must contain an additional variable once `stepBased` is chosen for `iterationEncoding`:
+
+  - `snapshot`
```

Comment:
Question: we could allow to skip this if only one iteration (snapshot) is written.
In that case, the implied value should be `0` and there must be exactly one update/step in the data format.

Reply:
This is what the other backends actually do when that iteration encoding is chosen, see the `variableBasedSingleIteration` test. The `snapshot` attribute is not written.

Reply:
With the current state of https://github.com/openPMD/openPMD-api/pull/949, the `snapshot` attribute is always written, but not required at read-time (then assumed to be 0). I should add a test somehow to ensure that reading without `snapshot` works as intended.

Reply:
now tested